File: _bench/fib/fib.java
import java.lang.System.*;

public class fib {
  public static void main(String[] args) {
    var n = 40;
    System.out.printf("%d\n", fib(n));
  }

  static long fib(long n) {
    if (n < 2) {
      return 1l;
    } else {
      return fib(n - 1) + fib(n - 2);
    }
  }
}

File: _bench/fib/fib.clj
(ns fib
  (:gen-class))

(set! *unchecked-math* :warn-on-boxed)

(defn fib ^long [^long n]
  (if (< n 2)
    1
    (+ (fib (- n 1)) (fib (- n 2)))))

(defn -main [& args]
  (let [n 40]
    (println (fib n))))
